packages/oc-template-react-compiler/lib/compileServer.js

```diff
@@ -12,7 +12,10 @@ const webpackConfigurator = require("./to-abstract-base-template-utils/webpackCo
 
 module.exports = (options, callback) => {
   const serverFileName = options.componentPackage.oc.files.data;
-  const serverPath = path.join(options.componentPath, serverFileName);
+  let serverPath = path.join(options.componentPath, serverFileName);
+  if (process.platform === "win32") {
```

Comment:
Good finding. In node, we have `path.sep` to provide a nice abstraction for path separators, so that we don't need to check the platform.
https://nodejs.org/api/path.html#path_path_sep
I wonder if we can use that to cleanup `options.componentPath` or `serverFileName` before?

Reply:
@matteofigus What's unique here is that the path is correct, however the backslashes need to be double escaped.  Thus ```\\\\``` vs ```\\```

If we can get around the need for double escaping then we won't need to check the os.

```
const higherOrderServerContent = `
    import { data as dataProvider } from '${serverPath}';
```